test/compflags/ferguson/check_ieee_float.chpl

```diff
@@ -0,0 +1 @@
+writeln("Hi");
```

Comment:
I prefer having 2 simple tests here because if there's a failure it's clear from the test name which failed. I've added the 2nd variants as compopts.
I'm not really excited about grepping the Makefile... I think these tests are more useful for checking that e.g. we're passing a legit flag to icc. The Makefile won't have the flag for icc in it.